backend/ApiGateway/ApiGateway/configuration.Production.json

```diff
@@ -71,6 +71,62 @@
       "UpstreamPathTemplate": "/api/Songs/search?searchTerm={searchTerm}",
       "UpstreamHttpMethod": [ "GET" ]
     },
+    {
+      "DownstreamPathTemplate": "/api/Songs/liked",
+      "DownstreamScheme": "https",
+      "DownstreamHostandPorts": [
+        {
+          "Host": "localhost",
+          "Port": "5001"
```

Comment:
Production config should not contain localhost. Please pay attention on other routes in this file

Reply:
Oh, sure, thanks